persistence-modules/spring-data-jpa-query/src/test/java/com/baeldung/exists/CarRepositoryIntegrationTest.java

```diff
@@ -19,6 +20,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = {Application.class})
+@TestPropertySource(properties = {"spring.jpa.show-sql=false"})
```

Comment:
Wouldn't it be better to set this property in the application(-test).properties?

Reply:
I've removed the property from individual tests and added it to application.properties.